### DISCUSSION TITLE
Filter match highlight not displaying exact color

### DISCUSSION THREAD

**Discussion by DrBlackBird** [KevinVandy/material-react-table#1038] [Q&A] [answered]
Hi everybody!

I am currently implementing material-react-table and I was wondering how to change the filter match highlight color. The documentation tells me to change the color of palette.warning.main [Documentation](https://www.material-react-table.com/docs/guides/global-filtering#filter-match-highlighting). The documentation also points out, that `with a shade of the theme.palette.warning.main color`. I found that not the exact color is displayed but in fact a slightly lighter version of the color i provided. Is it possible to disable that "shading" somehow, without creating the cell on my own?

Thanks!

**Comment by KevinVandy** [comment, marked as answer]
https://www.material-react-table.com/docs/guides/customize-components#mrt-theme-values

**Reply by DrBlackBird** [reply]
Thank you very much, that helps!! I will play around with it tomorrow but I think that will solve my issue. I need to find out where to inject the mrtTheme object but that might be just some documentation reading. :)

**Reply by KevinVandy** [reply]
It's just a table option

**Reply by DrBlackBird** [reply]
It did work, thank you :)

**Comment by githorse** [comment]
This is what I was looking for as well:

```
mrtTheme: {
  matchHighlightColor: 'my-custom-color-here'
}
```

@KevinVandy you might want to add a mention of this in the docs at https://www.material-react-table.com/docs/guides/global-filtering#filter-match-highlighting; the verbiage there made me think that the highlight color was not configurable, and it took me a while to find this discussion.
